library/control/src/modules/WorkflowManager.rb

```diff
@@ -452,21 +464,27 @@ def addon_control_dir(src_id, cleanup: false)
     # Returns requested control filename. Parameter 'name' is ignored
     # for Add-Ons.
     #
-    # @param [Symbol] type `addon or `pattern
+    # @param [Symbol] type :addon or :pattern or :package, pattern not implemented
```

Comment:
Remove `:pattern` from `:addon or :pattern or :package`, it's strange to name it and later write it's not supported.

Plus mention returning `nil` in that case. (Should be also added to `@return`.)